registration/src/main/java/com/kyriba/registration/configuration/SwaggerConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.kyriba.registration.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig
+{
+  @Bean
+  public Docket api()
+  {
+    return new Docket(DocumentationType.SWAGGER_2)
+        .select()
+        .apis(RequestHandlerSelectors.basePackage("com.kyriba.registration.api"))
```

Comment:
Add general information about API and apply filter for URLs.

Reply:
done